.mci.yml

```diff
@@ -168,7 +168,7 @@ functions:
                         PREFIX=$(cygpath -m "$PREFIX")
                     fi
                     export GENERATOR="${cdriver_cmake_generator}"
-                    ./.evergreen/install_c_driver.sh ${MONGOC_VERSION|master}
+                    ./.evergreen/install_c_driver.sh ${mongoc_version|master}
```

Comment:
I changed the case for consistency with other expansions. I've been bitten before when treating an expansion as a bash variable and vice-versa.

Reply:
Acknowledged